chapter_recurrent-neural-networks/language-models-and-dataset.md

```diff
@@ -181,7 +181,7 @@ which is equivalent to
 $$\log n_i = -\alpha \log i + c,$$
 
 where $\alpha$ is the exponent that characterizes the distribution and $c$ is a constant.
-This should already give us pause if we want to model words by count statistics and smoothing.
+This should already give us pause if we want to model words by counting statistics and smoothing.
```

Comment:
count is a noun here

Reply:
I found that couting statistics is a phrase, but count statistics is not.

Reply:
https://pubs.rsna.org/doi/pdf/10.1148/radiographics.19.3.g99ma33765

Reply:
You may also find "count statistics" in papers such as https://aapm.onlinelibrary.wiley.com/doi/abs/10.1002/mp.13063 :)

Reply:
I found that couting statistics had more usage than count statistics. Maybe count means the total number counted, but counting means the act of counting.

Reply:
![Screen Shot 2021-06-25 at 8 21 37 PM](https://user-images.githubusercontent.com/22279212/123500448-fc804a00-d5f2-11eb-9559-6c28d402da4d.png)

I feel that "count" serves this context better

Reply:
I could found that, in the above sentences,  "count" is the noun and used in the end of the phrase. The meaning of "count" is the total number counted. But "counting" in "counting statistics" maybe used for adjective, and it's meaning is the act of counting. So I feel that "counting" maybe the best choice for embellishing "statistics".

Reply:
Both nouns and adjectives are common modifiers. Nonetheless, I am leaning towards "counting statistics" after reading this paper: https://arxiv.org/pdf/cond-mat/9607137.pdf

Reply:
OK.
Glad to communicate with u.

Reply:
The whole sentence here is ungrammatical. I'll revise.